src/main/resources/TotemCode/WebPreferences.xml

```diff
@@ -0,0 +1,1262 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Could there not be some side effects of having a preference page for the space with a skin set. If it can be avoided that would be better